storagenode/storagenodedb/bandwidthdb_test.go

```diff
@@ -0,0 +1,39 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package storagenodedb
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+
+	"storj.io/storj/internal/testrand"
+	"storj.io/storj/pkg/pb"
+)
+
+func TestBandwidthdb_Trivial(t *testing.T) {
```

Comment:
These tests are near the interface not the implementation.

Reply:
I don't understand what this means. What's the desired change?